bxslider.js error

Please include the following information when filing an issue 
• Current version of FBCTF installed (commit head)
• System and Installation details, such as the Linux distribution used, and installation method you chose.
• Detailed description of the problem. Include steps to reproduce the problem when possible.
• Screenshots to help describe the problem when appropriate.
• Relevant output from any logs examined.
• Any steps you have taken to resolve the issue.

I am running the following command and getting a similar error:

sudo ./extra/provision.sh -m dev -s $PWD

Below is how my command ends before returning to the terminal prompt after several minutes:

Running "browserify:dist" (browserify) task
>> Error: Cannot find module 'bxslider' from '/var/www/fbctf/src/static/js'
Warning: Error running grunt-browserify. Use --force to continue.

Aborted due to warnings.

--- a/package.json
+++ b/package.json
@@ -26,6 +26,6 @@
     "jquery": "^2.2.3",
     "keycode": "^2.1.1",
     "typed.js": "^1.1.1",
-    "bxslider": "~4.2.6"
+    "bxslider": "4.2.6"
   }
 }

